tests/CMakeLists.txt

```diff
@@ -20,6 +20,28 @@ add_c_test(format_portability_unit)
 add_c_test(robust_deserialization_unit)
 add_c_test(container_comparison_unit)
 add_c_test(add_offset)
+if(CMAKE_GENERATOR MATCHES "Visual Studio")
+  message(STATUS "Visual Studio does not yet support C11 atomics.")
+  message(STATUS "https://www.technetworkhub.com/c11-atomics-in-visual-studio-2022-version-17/")
+endif()
```

Comment:
Probably don't need to write this message anymore, since we made an impl for visual studio too.

Reply:
Fixed.